### PR TITLE
Feat.eap links

### PR DESCRIPTION
**Pull request by jannisvisser**

### CODE REVIEW

**Comment by jannisvisser** [comment]
This PR contains a bunch of small things across items (see commit messages).
In terms of related content it is still awaiting. But this doesn't block from reviewing this already.
- some missing input for info popup texts ETH layers 
- EAP-link Zimbabwe

File: services/API-service/src/scripts/seed-init.ts [hunk 1]
```diff
@@ -106,7 +106,7 @@ export class SeedInit implements InterfaceScript {
           countryEntity.adminRegionLabels = JSON.parse(
             JSON.stringify(country.adminRegionLabels),
           );
-          countryEntity.eapLink = country.eapLink;
+          countryEntity.eapLinks = JSON.parse(JSON.stringify(country.eapLinks));
```
Comment (SwatiTh) [inline services/API-service/src/scripts/seed-init.ts:109]:
```suggestion
          countryEntity.eapLinks = JSON.parse(JSON.stringify(country.eapLinks));
          countryEntity.eapLinks = {...country.eapLinks} 
```

Comment (jannisvisser) [reply services/API-service/src/scripts/seed-init.ts:109]: @SwatiTh this suggestion gives me immediate red errors in VS code 

Type '{ floods: string; malaria?: undefined; "heavy-rain"?: undefined; dengue?: undefined; drought?: undefined; } | { floods: string; malaria?: undefined; "heavy-rain"?: undefined; dengue?: undefined; drought?: undefined; } | ... 4 more ... | { ...; }' is not assignable to type 'JSON'.
  Type '{ floods: string; malaria?: undefined; "heavy-rain"?: undefined; dengue?: undefined; drought?: undefined; }' is missing the following properties from type 'JSON': parse, stringify,

**Comment by SwatiTh** [comment]
All looks nice to me, just few minor things I came across mentioned here:
For ETH newly added Layer:
malaria:
Potential cases 65: Link provided in text is not hyperlinked(not clickable ).

9143:if first character will be in upper case it will look good:
under age:vulnerable population group Ethiopia: High Resolution Population Density Maps + Demographic Estimates.

**Pull request merged**